def validate(file_path):
    raise RuntimeError('Validation Successful!')
